src/app/main/login/login.component.ts

```diff
@@ -27,15 +28,26 @@
   }
 
   login(){
+    const HTTP_OPTIONS = {
+      headers: new HttpHeaders({
+        'Content-Type': 'application/x-www-form-urlencoded',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ticodacytest/chatApp1/pullRequest?prid=7784558)
```suggestion
        "Content-Type": 'application/x-www-form-urlencoded',
```